webapp/src/jaas/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+
+import store from './reducers/store';
+import JaaSConference from './components/conference';
+
+function JaaSRoot() {
+    return (<JaaSConference/>);
+}
+
+export class InjectionProvider extends React.Component<any> {
+    public render(): JSX.Element {
+        const stores = {...this.props};
+        delete stores.children;
+        return React.createElement(Provider as any, stores, this.props.children);
+    }
+}
+
+ReactDOM.render(<InjectionProvider store={store}>
+    <JaaSRoot/>
+</InjectionProvider>, document.getElementById('jaas-root'));
```

Comment:
@raghavaggarwal2308 Can you explain what's going on here?

Reply:
@mickmister For Jaas meeting we have used everything related to redux separate from that of mattermost. The main reason is that we don't want to load the entire mattermost redux state and use all of its functions just for the Jaas server meeting as this would be faster and will not reduce the performace.

Reply:
@Kshitij-Katiyar Are you saying we're using `ReactDOM.render` for the purpose of using a separate redux store? That seems like a bad antipattern if that's the case. Any alternatives to using `ReactDOM.render` here?

Reply:
On the topic of using `ReactDOM.render`, the plugin cannot assume this method is available. The API changed in react-dom 17, which the web app will start using in MM v7.7. Plugins should avoid calling methods on `ReactDOM`, except for methods like `createPortal`

Reply:
@mickmister In this plugin ReactDOM.render was already used to render the root portal that's why I used it in this PR as well.
https://github.com/mattermost/mattermost-plugin-jitsi/blob/f52f77ee59f80507809a6f472d68138c9dccad94/webapp/src/components/root_portal.tsx#L48